src/main/java/org/zalando/nakadi/service/EventTypeService.java

```diff
@@ -180,52 +181,50 @@ private void validateSchemaChange(final EventType eventType, final EventType exi
         }
     }
 
-    private void validatePartitionKeys(final EventType eventType) throws InvalidEventTypeException {
```

Comment:
The problem with removing this toggle is that currently it's impossible to change schemas but it's possible to change partition keys. But after merging this code it will be impossible to change partition keys if they are not defined in the schema.

But anyway, I think if we notify users properly, there should be no major consequences.